pkgs/by-name/cy/cyme/package.nix

```
@@ -40,7 +48,14 @@ rustPlatform.buildRustPackage rec {
     "--skip=test_run"
   ];
 
-  passthru.updateScript = nix-update-script { };
+  passthru = {
+    updateScript = nix-update-script { };
+    tests.version = testers.testVersion {
+      package = cyme;
```

```suggestion
      package = finalAttrs.finalPackage;
```

If you chose to opt for the `finalAttrs` pattern, not a blocker.